react-components/src/components/Reveal3DResources/types.ts

```diff
@@ -108,4 +108,5 @@ export type CommonResourceContainerProps = {
   image360Settings?: CommonImage360Settings;
   onResourcesAdded?: () => void;
   onResourceLoadError?: (failedResource: AddResourceOptions, error: any) => void;
+  onCallback?: (data?: any) => void;
```

Comment:
Can this have a more descriptive name?

Reply:
I was thinking to use a more generic name for other purposes any other than rule base callback. 
Because it can be used to anything else if we isolate this component into some other context.

Reply:
changed here
https://github.com/cognitedata/reveal/pull/4652/commits/01d9e4192802dde46298958dce3453c03f561cce